Mods/ArchipelagoPatches.SC2Mod/enUS.SC2Data/LocalizedData/TriggerStrings.txt

```diff
@@ -0,0 +1,3 @@
+﻿CustomScript/Name/lib_6EE41A55_F0ECB691=Untitled Custom Script 001
+Library/Name/6EE41A55=ArchipelagoPatches
+Trigger/Name/lib_6EE41A55_59D28101=Untitled Trigger 001

```

Comment:
I don't see them appear anywhere in the GUI trigger editor; all the names should be visible in the top-left panel of my last image.

Reply:
![Snímek obrazovky pořízený 2024-02-17 01-20-22](https://github.com/Ziktofel/Archipelago-SC2-data/assets/3447808/5f6838b2-32a0-4bac-a3ff-5def151b2341)
it's in the Patches mod after I've opened it

Reply:
![Snímek obrazovky pořízený 2024-02-17 01-20-22](https://github.com/Ziktofel/Archipelago-SC2-data/assets/3447808/ff6c7466-d307-42bf-9614-5f4470ef57f5)
This is what I get when I open the patches mod. Looks like you haven't committed a file properly

Reply:
No, the file is in the same state for me locally as in this commit, and I still get the correct naming.

![image](https://github.com/Ziktofel/Archipelago-SC2-data/assets/31861583/7607ea47-4e01-4c08-b80b-6634bc5c5b06)
![image](https://github.com/Ziktofel/Archipelago-SC2-data/assets/31861583/ea355672-48e8-4930-a44f-9987e6b7d8c1)

I'll update TriggerStrings.txt manually and commit again, but I can't verify that does anything because the problem isn't appearing on my computer.

Reply:
![Snímek obrazovky pořízený 2024-02-18 12-38-38](https://github.com/Ziktofel/Archipelago-SC2-data/assets/3447808/9c723c9f-1cb0-42b0-91c5-be3009c51351)
That did the trick (just did a git pull and opened the mod in the Editor)